apex-staging.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 scrnaseq:
   label: "Single-cell RNA processing"
-  version: "v1.0.0rc3"
+  version: "v1.0.0rc4"
   description: "MultiQC and sample HTML reports"
   workspace_id: "236559192150132"
   compute_env_id: "2GelmR4XvPuTuQcrB9CqRG"
```

Comment:
Please update this to `uEC2MMC1WSJaolYipyNpG` so we can begin to use the new Nextflow buckets